### PR TITLE
Added quotation marks to automod phrases

### PR DESCRIPTION
**Pull request by hicupalot**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description
Added quotation marks around the automod phrases when they are added/removed from filter so it'll read like:
User added "phrase" to automod rather then User added phrase to automod. This allows for it to be far clearer

### CODE REVIEW

**Comment by hicupalot** [comment]
Sorry I did that now!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## Unversioned
-
+- Minor: Added quotation marks around the automod phrases for clairty 
 ## 2.3.5
```
Comment (Felanbird) [inline CHANGELOG.md:5]:
```suggestion
- Minor: Added quotation marks in the permitted/blocked Automod messages for clarity. (#3654)

## 2.3.5
```
Also no worries on the ✅ - it's basically cosmetic and maintainers can press it if they care enough.

Comment (hicupalot) [reply CHANGELOG.md]: Thanks for clearing that up!

**Comment by hicupalot** [comment]
It said it failed at the last check, is there something I need to do to resolve this

**Comment by Felanbird** [comment]
> It said it failed at the last check, is there something I need to do to resolve this

`./tools/clang-format-all.sh` and commit the formatted changes

**Comment by hicupalot** [comment]
> > It said it failed at the last check, is there something I need to do to resolve this
> 
> `./tools/clang-format-all.sh` and commit the formatted changes

Ran it and nothing seems to have happened and it wont let me commit anything as there were no changes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unversioned
+- Minor: Added quotation marks in the permitted/blocked Automod messages for clarity. (#3654)
 
 - Bugfix: Fixed live notifications for usernames containing uppercase characters. (#3646)
```
Comment (Felanbird) [inline CHANGELOG.md:6]:
```suggestion
## Unversioned

- Minor: Added quotation marks in the permitted/blocked Automod messages for clarity. (#3654)
- Bugfix: Fixed live notifications for usernames containing uppercase characters. (#3646)
```

Comment (Felanbird) [reply CHANGELOG.md:6]: I didn't notice it was the linter that was upset, not the format check

**Review by Felanbird — APPROVED** [review]
Just a minor QoL - since our messages are structured where the term is in the middle of the message, in comparison to twitch chat where it's at the very end, this seems like a perfectly fine change for clarity.

**Comment by hicupalot** [comment]
> Just a minor QoL - since our messages are structured where the term is in the middle of the message, in comparison to twitch chat where it's at the very end, this seems like a perfectly fine change for clarity.

Thank you!

**Comment by hicupalot** [comment]
Thank you for all your help! The checks passed 😄

**Review by Mm2PL — APPROVED** [review]
Simple change but it should really improve the readability of the automod messages.

**Comment by Felanbird** [comment]
Thanks for your contribution! As a first-time contributor, you can now add yourself to the contributors list that's shown inside the application.

To do so, open a new PR where you modify the resources/contributors.txt file and add yourself to the list. Make sure to read the comments at a top for instructions.

**Pull request merged**